packages/language-server/test/ui5-model-spec.ts

```diff
@@ -15,7 +15,7 @@ import { forEach, isPlainObject } from "lodash";
 
 describe("the UI5 language assistant ui5 model", () => {
   // The default timeout is 2000ms and getSemanticModel can take ~3000-5000ms
-  const GET_MODEL_TIMEOUT = 10000;
+  const GET_MODEL_TIMEOUT = 20000;
```

Comment:
Is the increase timeout because your internet connection is slow or because the usage of the proxy agent causes a performance regression?

Reply:
i'll recheck this, it was especially when i tried to use axios, might not need the increase anymore.